_content/articles/nature-deities-of-tibet_rikey-thupten-k.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Nature-Deities of Tibet: A discussion on the tale “The Subduing and Putting under Oath of Tibet’s Malignant lha 'dre” in Padma bka' thang"
+authors:
+  - "Thupten K. Rikey"
+external_url: "https://journal.fi/store/article/view/51549"
+drive_links:
+  - "https://drive.google.com/file/d/1BaRKwu6SuGUO_8oz-8YmfZpZC03unT1V/view?usp=sharing"
+course: tibetan
+tags:
+  - tantric
+year: 2011
+journal: store
+volume: 109
+pages: "119--140"
+---
+
+>Among the nature-deities the tale under discussion depicts as confronting
+Padmasambhava, two (see below) are among the nine deities known as bod kyi
+srid pa chags pa'i lha, meaning the original deities of Tibet, whose function is to
+protect the land and people of Tibet.
+
+This paper analyzes the depiction of nature-deities in the 14th-century Padma bka'thang, focusing on the supernatural conflict between Padmasambhava and Tibetan deities resisting his arrival. It interprets this confrontation as a reflection of deeper cultural and religious dynamics during Tibet's early Buddhist era.
```

Comment:
"Padma bka'thang" is a book, right?  Book titles get *s around them (and articles, etc get "s)